### ISSUE TITLE
How to test XMR-STAK cryptonightR version?

### ISSUE THREAD

**Issue by kio3i0j9024vkoenio** [fireice-uk/xmr-stak#2266]
I want to test the upcoming cryptonightR version of XMR-STAK on my various Intel and AMD servers before the fork happens.

I have found this link: https://github.com/fireice-uk/xmr-stak/tree/feature-cryptonightR

but trying to clone I see this link: https://github.com/fireice-uk/xmr-stak.git
That doesn't appear to be the cryptonightR version.

It also appears that downloading the ZIP results in the V8 version not the cryptonightR version.

Where can I download (clone) the cryptonightR version?

**Comment by Spudz76** [comment]
Yes you checkout from the same repository so the remote URL is the same
once you have the tree checked out you will be in `master` branch by default

just do `git checkout feature-cryptonightR` from there
OR technically you can put `#feature-cryptonightR` on the end of the clone URL and it will checkout directly to that instead of master at all

`git status` after such should say you are on that branch
`git diff origin/dev` would show you the diffs compared to what `dev` branch currently contains
`git log` should show the commits beyond where it says the `dev` branch is at

```
commit 82732c20979cfb03033430970cbd2d9345339e0e (HEAD -> feature-cryptonightR, origin/feature-cryptonightR)
Author: psychocrypt <psychocryptHPC@gmail.com>
Date:   Sun Feb 17 19:52:33 2019 +0100

    add cryptonight_r code generator

commit 204336e990118a1d668e04e49f41eb6a4afa4167
Author: psychocrypt <psychocryptHPC@gmail.com>
Date:   Sun Feb 17 19:05:38 2019 +0100

    add `cryptonight_r*` definitions

    Add curency/algorithm definition for
     - cryptonight_r
     - cryptonight_r_64
     - cryptonight_r_wow

commit cc2a683fc5399be5dc65103dd09153313a52f84c (origin/dev, dev)
<snip>
```

**Comment by psychocrypt** [comment]
I will hopefully on sunday a version ready.
